contracts/Liquidator/Liquidator.sol

```diff
@@ -363,19 +360,21 @@ contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, Liqu
         uint256 range = _pendingRedeemLength >= pendingRedeemChunkLength
             ? pendingRedeemChunkLength
             : _pendingRedeemLength;
-        if (range == 0) return;
-        for (int256 index = int256(range) - 1; index >= 0; index--) {
-            address vToken = pendingRedeem[uint256(index)];
+        for (int256 index = int256(range); index > 0; ) {
```

Comment:
I think now we can use uint instead of int256, avoiding the casting

Reply:
Fixed [here](https://github.com/VenusProtocol/venus-protocol/pull/298/commits/a2e94c94e0aedf478a8bcee1efdecb1958bbe20c)